### ISSUE TITLE
command-execute: Wrong type argument: commandp, debug-help-follow

### ISSUE THREAD

**Issue by sarg** [emacs-evil/evil-collection#231]
I'm using emacs 27.0.50 and this binding in `evil-collection-debug-setup` isn't working. When I am in `*Backtrace*` buffer, pressing `RET` on function name in backtrace raises error. Maybe use default `backtrace-help-follow-symbol`?

```elisp
(defun evil-collection-debug-setup ()
...
    (kbd "<return>") 'debug-help-follow
```

**Comment by jojojames** [comment]
I don't have the motivation to build/reproduce/fix/test on emacs 27 when it's not yet released so a PR with a compability check (if emacs 27 or emacs 26) would be very welcome.

**Comment by sarg** [comment]
Fixed in: 5d32ceb
